### ISSUE TITLE
AWS::ECT::TaskDefinition properties not being recognized

### ISSUE THREAD

**Issue by rmsilva1973** [aws-cloudformation/cfn-lint#2745]
### CloudFormation Lint Version

cfn-lint 0.77.5

### What operating system are you using?

Ubuntu on WSL

### Describe the bug

None of the valid properties for AWS::ECS::TaskDefinition are being reconized as can be seen on the screenshot below. I tried runnning ```cfn-lint --update-specs``` and restarting VSCode but still nothing.

![image](https://github.com/aws-cloudformation/cfn-lint/assets/28487379/05400708-2d02-4632-8eb2-600350298c59)

### Expected behavior

cfn-lint should not flag this properties as errors as they are valid for this CFN resource.

### Reproduction template

```yaml
  MyTask:
    Type: 'AWS::ECS::TaskDefinition'
    Properties:
      Environment:
        - Name: HOST
          Value: !Ref MyHostUrl
      Essential: true
      HealthCheck:
        Command:
        - CMD-SHELL
        - curl -f http://127.0.0.1:8080/actuator/info || exit 1
        Interval: 200
        Retries: 3
        StartPeriod: 100
        Timeout: 60
      Image: !GetAtt Repository.RepositoryUri

**Comment by rmsilva1973** [comment]
cfn-lint is working properly. My CFN template is incorrect. 🙈
